.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
Was this added accidentally :0 ?

Reply:
good catch, I believe it should be fixed now